docs/src/main/paradox/release-notes/10.2.x.md

```diff
@@ -26,10 +26,15 @@ we release 10.2.0 proper.
 * Silence outgoing request stream error [#2905](https://github.com/akka/akka-http/pull/2905)
 * Add SameSite attribute to Cookies [#2928](https://github.com/akka/akka-http/pull/2928)
 * Only catch NonFatal Exceptions [#2853](https://github.com/akka/akka-http/pull/2853)
+* Add coordinated shutdown support [#3142](https://github.com/akka/akka-http/pull/3142)
 * Percent-encode illegal chars when creating URI query [#3003](https://github.com/akka/akka-http/pull/3003)
 * Add the remote address in parse errors when possible [#2899](https://github.com/akka/akka-http/pull/2899)
 * Remove `UseHttp2` [#2896](https://github.com/akka/akka-http/pull/2896)
+* Hide body and headers by default in `HttpRequest#toString` and `HttpResponse#toString` [#2560](https://github.com/akka/akka-http/pull/2560)
 * Fix headers javadsl scaladoc [#2932](https://github.com/akka/akka-http/pull/2932)
+* Allow client setting overrides for target hosts in config [#2574](https://github.com/akka/akka-http/pull/2574)
```

Comment:
Maybe put that one into the headline features. It had been breeding for long and is a really useful feature.